packages/repository/test/unit/repositories/constraint-utils.unit.ts

```diff
@@ -7,14 +7,14 @@ import {expect} from '@loopback/testlab';
 import {
   FilterBuilder,
   Filter,
-  Where,
   WhereBuilder,
   constrainFilter,
   constrainWhere,
   constrainDataObject,
   constrainDataObjects,
   Entity,
 } from '../../..';
+import {AnyObject} from '../../../src';
```

Comment:
Please import from `../../..`, not `../../../src`.

Also join this import statement with the one on the previous line. Ideally, run the task "Organize imports" in VSCode.